tests/sanity/ignore-2.10.txt

```diff
@@ -5,3 +5,5 @@
 .azure-pipelines/scripts/publish-codecov.py future-import-boilerplate
 .azure-pipelines/scripts/publish-codecov.py metaclass-boilerplate
 plugins/modules/acme_account_info.py validate-modules:return-syntax-error
+tests/ee/roles/smoke/library/smoke_ipaddress.py shebang
+tests/ee/roles/smoke/library/smoke_pyyaml.py shebang
```

Comment:
Did you see my comment in a previous one of these?
I don't know the reason why (maybe it's because they are in an integration target? or a `setup_` target specifically?), but I do not have any shebang errors on modules I created for tests:
https://github.com/ansible-collections/community.hashi_vault/tree/main/tests/integration/targets/setup_vault_test_plugins/library
(some have a shebang, I think most don't, they all work)

Reply:
The reason is that this isn't in tests/integration/targets/.../library/ or plugins/modules/, but some other directory. The sanity check treats these directories specially: https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_util/controller/sanity/code-smell/shebang.py#L84